src/shared/hooks/use-required-question.tsx

```diff
@@ -1,18 +1,17 @@
 import React, { useEffect } from "react";
-
-interface IConfig {
-  authoredState: { required?: boolean } | undefined;
-  interactiveState: { submitted?: boolean } | undefined;
-  setNavigation: ((enableForwardNav: boolean, message: string) => void) | undefined;
-}
+import { setNavigation, useAuthoredState, useInteractiveState, useInitMessage } from "@concord-consortium/lara-interactive-api";
 
 // This hook can be used by any interactive that defines `required` property in its authored state and
 // `submitted` property in its interactive state (student state).
-export const useRequiredQuestion = ({ authoredState, interactiveState, setNavigation }: IConfig) => {
+export const useRequiredQuestion = () => {
```

Comment:
If the submit button took the interactiveState type and authoredState type, they could be passed into this hook as generics as well to provide some type checking to verify what is described in the comment on line 4 and 5.